### DISCUSSION TITLE
How do I put HTML entity inside view!{}?

### DISCUSSION THREAD

**Discussion by trevorwslee** [leptos-rs/leptos#2299] [Q&A]
It appears to me that the following &mdash; will just be rendered as "&mdash;", not a — that I want
view! {
  &mdash;
}
Please help to hint the correct way to put HTML entity inside view!{}
Thanks.

**Comment by gbj** [comment]
You can't, because these are used to set the `textContent` of `Text` nodes directly in the browser, which doesn't involve HTML parsing, in addition to being used for HTML generation on the server.

You can, however, use ordinary Unicode in strings.
